package.json

```diff
@@ -27,6 +27,7 @@
 		"@typescript-eslint/eslint-plugin": "^6.7.5",
 		"@typescript-eslint/parser": "^6.7.5",
 		"autoprefixer": "^10.4.16",
+		"envsafe": "^2.0.3",
```

Comment:
It's used during runtime so I think it's a dependency, not a devDependency.

Reply:
It's actually used during setup, that's why I put it in devDependency. We don't need to build this with the final bundle